test/shopify-cli/context_test.rb

```diff
@@ -10,6 +18,42 @@ def setup
       @ctx = Context.new
     end
 
+    # rubocop:disable Minitest/TestMethodName
+    def with_stubbed_context(&block)
+      @old_config = ShopifyCLI::Config
+      without_warnings do
+        ShopifyCLI.const_set(:Config, Class.new do
```

Comment:
`const_set` here and below displays a warning, since the constant has already been set. So I disable warnings here to avoid unhelpful output in tests.